source/search_page/assets/script.js

```diff
@@ -23,4 +21,29 @@ function createRecipeCards() {
     
   
 }
+*/
+
+async function searchFetchRecipes(searchBar) {
+  let searchString = `https://api.spoonacular.com/recipes/complexSearch?apiKey=03722052291e4f84bce1021acd82624f&${searchBar}&addRecipeInformation=True`;
+  console.log(searchString);  
+  return fetch(searchString)
+    .then((response) => response.json());
+}
+
+window.onload = async function(){
+  let questionLocation = window.location.href.indexOf("?");
+  if (questionLocation < 0 ){
+    return;
+  }
+  let querystring = window.location.href.substring(questionLocation + 1);
+  let resultArray = await searchFetchRecipes(querystring).then((response) => {
+    return response.results;
+  });
+  for(let i=0; i< resultArray.length; i++){
+    let ele = document.createElement('search-recipe');
+    ele.data = resultArray[i];
+    document.querySelector('main').append(ele);
+
+  };
```

Comment:
Codacy has a fix for the issue: [Unnecessary semicolon.](https://app.codacy.com/gh/Big-Bytez/cse110-fa21-group19/pullRequest?prid=8435053)
```suggestion
  }
```